tests/zfs-tests/tests/functional/cli_root/zfs_unmount/zfs_unmount_nested.ksh

```diff
@@ -45,20 +45,24 @@ function nesting_cleanup
 log_onexit nesting_cleanup
 
 set -A test_depths 30 16 3
+typeset mountpoint=/$TESTPOOL/mnt
```

Comment:
Getting the mountpoint after the initial dataset creation (line 50) would be a little more robust.

```sh
typeset mountpoint="$(get_prop mountpoint $TESTPOOL/$dsA32)
```

Reply:
The mountpoint is supposed to be in the root of the pool though, that would get the mountpoint of a deeply nested dataset.

Reply:
Ahh, OK.  My mistake.